pillar/top.sls

```diff
@@ -1,4 +1,4 @@
-base:
+{{ saltenv }}:
```

Comment:
Based on pillar/common, this will always be `base` correct?

Reply:
No, for pillar it will be whatever the configured saltenv is